design/ceph/object/ceph-rgw-support-multisite-for-external-cluster.md

```diff
@@ -0,0 +1,31 @@
+---
+# Support Realm/Zone/Zonegroup for external RGW server
+target-version: release-X.X
+---
+
+# Feature Name
+ Support Realm/Zone/Zonegroup for external RGW server
+
+## Summary
+Rook supports handling for external RGW servers by assuming it is part of `default` zone/zonegroup. To handle consistently it is better handle this info, otherwise the radosgw apis may fail for such RGW servers.
```

Comment:
Where does rook assume the default zone/zonegroup?

Reply:
Please refer [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/objectstore.go#L198)